.github/workflows/macos-executable-release.yml

```diff
@@ -41,7 +60,7 @@ jobs:
         APPLE_ID_APP_PASSWORD: ${{ secrets.APPLE_ID_APP_PASSWORD}}
         APPLE_TEAM_ID: ${{ secrets.APPLE_TEAM_ID}}
 #        FILE_PATH_ARM64: dist/macos/${{ github.event.inputs.formula }}-v${{ needs.get-tag.outputs.TAG_NAME }}-${{ needs.get-sha.outputs.SHA_SHORT }}-arm64.pkg
-        FILE_PATH_X64: dist/macos/${{ github.event.inputs.formula }}-v${{ needs.get-tag.outputs.TAG_NAME }}.pkg
+        FILE_PATH_X64: ${{ needs.set-file-path.outputs.FILE_PATH_X64 }}
```

Comment:
As you can see from comment on #62, we also support arm64 files. Right now because of oclif upgrade we haven't been able to support it, but we should. Can you also add this logic for arm64 files?

Reply:
Added